examples/teleport-usage/README.md

```diff
@@ -0,0 +1,38 @@
+# Teleport Usage Gathering Script
+
+This script retrieves the number of unique users accessing each of the five Teleport supported protocols.
+
+## Prerequisites
+
+This tool requires a Teleport cluster running with AWS DynamoDB as your backend server. This script runs as a docker container from either the auth server or a server with IAM permissions necessary to run queres on the DynamoDB events table. 
+
```

Comment:
Proper permissions for DynamoDB require a minimum of:

"dynamodb:Get*", "dynamodb:Query", "dynamodb:Scan"

or they can use default AWS policy AmazonDynamoDBReadOnlyAccess